### ISSUE TITLE
adição de nova tag 'Cybersecurity'

### ISSUE THREAD

**Issue by ProgramadorVictor** [levxyca/diciotech#214]
### Descrição do recurso

'Cybersecurity': eu adicionei essa tag, pois inseri conceitos de segurança de software para alguns termos, como XSS e CSRF. Possivelmente, alguém vai adicionar termos relacionados e incluir a tag para outros termos, como SQL Injection e phishing, entre outros.

**Comment by george-gca** [comment]
Na verdade já tem essa tag, mas tá em português: `Segurança cibernética`.

**Comment by levxyca** [comment]
@george-gca bah, podiamos ter uma forma menos manual de trabalhar essas tags né? 🤔

**Comment by george-gca** [comment]
Eu concordo. Tens algo em mente?

O ideal seria algo que rodasse automaticamente e editasse o markdown. Ou até mesmo tirar as tags do arquivo, não sei se elas acrescentam muito estando aí.

**Comment by levxyca** [comment]
@george-gca hmmmm 🤔 preciso pensar melhor sobre isso, de imediato, a coisa mais simples que consigo pensar é dividirmos os termos técnicos em arquivos pelas categorias das tags, então por exemplo, ter um arquivos pra tudo que for de cybersec, outro front-end e assim vai, acho que assim já facilitaria um pouco mais a pessoa teria que adicionar dentro de alguma dessas opções de arquivos existentes, acredito ser mais simples do que como está hoje, mas não sei também 😵‍💫

**Comment by george-gca** [comment]
Acho que é melhor dividir por letra mesmo, porque vários termos vão aparecer em várias tags, aí fica aquela dúvida: em qual tag botar o termo?

**Comment by levxyca** [comment]
@george-gca verdade, tem isso né, um termo pode estar dentro de mais de uma categoria, mas nesse caso, será que não vale dai esse termo ir pra uma categoria mais abrangente como "conceito" não sei 🤔 definitivamente precisamos de melhorias nessa organização 🥹

**Comment by george-gca** [comment]
Eu acho que por letra é melhor, evita inclusive esse tipo de discussão, até porque o nome do termo já é a "chave" dele. O único "contra" de usar por letras é se botarmos suporte a múltiplos idiomas, que por exemplo `Deep neural network` ia ficar no D enquanto `Rede neural profunda` no R, mas nem digo que sou contra, só digo que a pessoa que for atualizar o termo em outro idioma tem que botar na letra certa também.

**Comment by levxyca** [comment]
@george-gca concordo, acho que a solução que mais se encaixa no momento é por letra mesmo.
